docker/README.md

```diff
@@ -0,0 +1,83 @@
+--------
+# Docker images and builders for BigDL
+
+## BigDL in Docker
+
+### By default, the BigDL image has installed below packages:
+* git
+* maven
+* Oracle jdk 1.8.0_152 (in /opt/jdk1.8.0_152)
+* python
```

Comment:
python version?

Reply:
Python 2.7.6
Python 3.4.3

I will add this on.